platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/ECMSDocumentActivityStreamTestIT.java

```diff
@@ -0,0 +1,87 @@
+package org.exoplatform.platform.qa.ui.platform.ecms;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.commands.Click;
+import com.codeborne.selenide.commands.Should;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.DocumentManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.ActivityStream;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Condition.not;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.refresh;
+import static org.exoplatform.platform.qa.ui.core.PLFData.*;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.ELEMENT_DATE_ACTIVITY;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ecms.ECMSLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+/**
+ * Created by esmaoui on 23/03/2018.
+ */
+
+@Tag("sniff")
+@Tag("ecms")
+public class ECMSDocumentActivityStreamTestIT extends Base{
```

Comment:
you can add this TC in SOCActivityAddTestIT, should not create class for each TC.